editoast/src/views/timetable/stdcm/request.rs

```diff
@@ -102,15 +104,16 @@ pub(super) struct Request {
     #[serde(default)]
     #[schema(value_type = Option<String>, example = json!(["5%", "2min/100km"]))]
     pub(super) margin: Option<MarginValue>,
-    /// Total mass of the consist in kg
-    #[validate(range(exclusive_min = 0.0))]
-    pub(super) total_mass: Option<f64>,
+    /// Total mass of the consist
+    //#[validate(range(exclusive_min = 0.0))] TODOUOM
```

Comment:
I suppose `Validate` isn't happy with the new types ^^

You could do the `remote = Self` trick to validate just after serializing: https://yossarian.net/til/post/remote-self-derivation-with-serde-for-custom-invariants/

Reply:
Validate somehow expected serializable structs and I didn’t find how to make him happy (I didn’t search much).

The `remote = "Self"` trick is funny, but it changes a bit the behavior: it would be a serialization error, and not explicitly in https://github.com/OpenRailAssociation/osrd/blob/dev/editoast/src/views/timetable/stdcm.rs#L144

Here is a suggestion: https://github.com/OpenRailAssociation/osrd/pull/9928/commits/abf6b1b419ae451abf79873eb7c30b8db1203232 what do you think?

Reply:
Lgtm, resolving. It's fine to keep using `validate` types since we can't get rid of the crate just yet anyways :)